### ISSUE TITLE
Loss functions

### ISSUE THREAD

**Issue by alexgkendall** [lixx2938/MegaDepth#19]
Awesome work, this is a fantastic concept. I had a question about the loss:

https://github.com/lixx2938/MegaDepth/blob/f1ffe4ca8b6c49e55e36259ac34909de6c90a27b/models/HG_model.py#L116

Why do you have a square root term here, this doesn't correspond to Eq. 2 in the paper?
For the hyperparams you supply in the paper are they with this sqrt loss or the one from your paper?

It would also be great to see implementations of the gradient loss term too if you can.

Thanks!

**Comment by zhangxgu** [comment]
I also want to see the implementations of the three loss in  paper...
